package.json

```diff
@@ -3,8 +3,10 @@
   "version": "3.2.1",
   "description": "Some tools for docker",
   "main": "lib/docker-toolbelt.js",
+  "types": "lib/docker-toolbelt.d.ts",
   "scripts": {
     "prepublish": "npm run test && coffee -c lib",
```

Comment:
You need to make this `prepublishOnly` and use `require-npm4-to-publish` to avoid issues with publishing on different npm versions

Reply:
I thought it would be fine with the circle config, but makes sense to harden this a bit, done.

Reply:
I'm think the circle config will actually ignore the `prepublish` script altogether because of its npm version

Reply:
Yeah I thought that, but it will still call the build step and the tests will have already been run 6 times before this, so it should be fine. I can add another test step to the circle deploy task if you like?